docs/source/ops/metering.rst

```diff
@@ -0,0 +1,94 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Participant Metering
+====================
+
+Participant metering is a way to report how many events have been submitted in a given period of time.
+
+Daml command execution results in a Daml transaction that contains events associated with the processing of the command.
+
+The events included in the report include:
+
+    *  Contract creation
+    *  Consuming exercise by contract identifier
+    *  Non-consuming exercise by contract identifier
+    *  Consuming contract exercise by contract key
+    *  Non-consuming contract exercise by contract key
+    *  Contract fetch by contract identifier
+    *  Contract fetch by contract key
+    *  Contract lookup by contract key
+
+Note that only events that originated from the local participant are included in the metering.  Events received
+by the local participant from remote participants are *not* included.
+
+Generating a Metering Report
+----------------------------
+
+A metering report is generated using the :doc:`Daml assistant </tools/assistant>` utility.
+
+To run a metering report ``daml ledger metering-report`` is used with the following metering specific arguments:
+
+:--from:
+    The period being reported on will start from the given date. Events on or after this date will be included.
```

Comment:
```suggestion
    A start date that is used to initiate the reporting period. Events on or after this date will be included.
```
To create parallel construction with the end date description.

Reply:
Done.